.eslintignore

```diff
@@ -1 +1,3 @@
-src/app/calc/evaluate_lua/__generated__
\ No newline at end of file
+src/app/calc/evaluate_lua/__generated__
+
+src/app/calc/__generated__/*.ts
```

Comment:
Added this since it will keep the formatting of these consistent with the output of the script, meaning clearer diffs.